requirements.txt

```diff
@@ -4,4 +4,5 @@ flake8
 pep8-naming
 lxml<=4.3.4
 psycopg2==2.7.3.1
-setuptools<58.0
+setuptools>=61.0.0 ; python_version > '2.7'
+setuptools<58.0 ; python_version <= '2.7'
```

Comment:
Where is this `requirements.txt` file used? 

I find a reference to it in in `.travis.yml` but that is obsolete.

Reply:
I don't know. That's why I'm asking for it, as I don't use pip deployment as you know, and don't know how this should be deployed. Does `pyproject.toml` serve  for all the Python versions, including Python 2.7?

Reply:
Just remove `requirements.txt` (it was not used before and is not used in this PR either), and `.travis.yml` while you are at it.

> Does pyproject.toml serve for all the Python versions, including Python 2.7?

Yes, pip 20.3.4 (the last version that supports python2) understands pyproject.toml.

Reply:
Done.